velox/scripts/config.sh

```diff
@@ -4,4 +4,4 @@
 CONTAINER_NAME="velox-adapters-build"
 COMPOSE_FILE="../docker/docker-compose.adapters.build.yml"
 
-NUM_THREADS=${NUM_THREADS:-$(( ($(nproc) > 2 ? $(nproc) - 2 : 1) ))}
```

Comment:
Fallback to 3/4th, to avoid CI lost communication with runner node error.

Reply:
I'm not sure about that.  If we use total_cores - 2 or total_cores, CI server lost communication with the runner machine.